lib/coinbase/wallet.rb

```diff
@@ -123,12 +123,15 @@ def wallets_api
     #   with the Wallet. If not provided, the Wallet will derive the first default address.
     # @param client [Jimson::Client] (Optional) The JSON RPC client to use for interacting with the Network
     def initialize(model, seed: nil, address_models: [])
-      validate_seed_and_address_models(seed, address_models)
+      validate_seed_and_address_models(seed, address_models) unless Coinbase.use_server_signer?
 
       @model = model
-      @master = master_node(seed)
       @addresses = []
-      @private_key_index = 0
+
+      unless Coinbase.use_server_signer?
+        @master = master_node(seed)
+        @private_key_index = 0
+      end
 
       derive_addresses(address_models)
```

Comment:
Can we add an `.import` test with server signer true that we properly load the addresses?

Reply:
done